sentry-android-ndk/proguard-rules.pro

```diff
@@ -1,6 +1,10 @@
 ##---------------Begin: proguard configuration for NDK  ----------
 
--keep class io.sentry.android.ndk.** { *; }
+# The Android SDK checks at runtime if this class is available via Class.forName
+-keep class io.sentry.android.ndk.SentryNdk { *; }
+# The JNI layer uses this classes through reflection
```

Comment:
Good catch, do we need to keep classes with `native` methods such as `NativeScope`?

Reply:
We already do this here below:
https://github.com/getsentry/sentry-java/blob/426d9be10fb3b28f16e73000834a686e1d755ae3/sentry-android-ndk/proguard-rules.pro#L10-L12

Reply:
oh yeah indeed.